Sources/WebDriver/WebDriver.swift

```diff
@@ -13,10 +13,15 @@ public protocol WebDriver {
 
 extension WebDriver {
     /// status - returns WinAppDriver status
-    /// Returns: an instance of the Status type
+    /// Returns: an instance of the WebDriverStatus type
     public var status: WebDriverStatus {
-        get throws { try send(Requests.Status()) }
+        get throws {
+            switch wireProtocol {
+                case .legacySelenium: return try send(Requests.Status_Legacy())
```

Comment:
Why `legacySelenium`? Why not `selenium` or `legacy`?

Reply:
Selenium refers it as its "Legacy JSON Protocol" and it's proprietary of Selenium instead of W3C standardized.

Reply:
I standardized on "LegacySelenium" all over.